src/modules/design/property-panel/indextabs/general/sections/service-task/service-task.ts

```diff
@@ -13,7 +13,7 @@ export class ServiceTaskSection implements ISection {
   public canHandleElement: boolean = false;
   public businessObjInPanel: IServiceTaskElement;
   public model: IPageModel;
-  @observable public selectedKind: string;
+  public selectedKind: string;
```

Comment:
You could introduce a better type for selectedKind like`type ServiceKind = '' | 'HttpClient' || 'external'`.

Or better yet an enum.:
```ts
enum ServiceKind {
   None = 'none',
   HttpClient = 'httpClient',
   External = 'external',
}
```
Bind it to the view with:
```ts
public ServiceKind: typeof ServiceKind = ServiceKind;
```
And access it like:
```html
<option value.one-time="ServiceKind.HttpClient">HTTP REST Service</option>
```

Reply:
Thought the html-attribute would need to have `"`-quotes, but seems to work just fine 👍

Just one more thing:
```suggestion
  public selectedKind: ServiceKind;
```
And then there's also this line in `activate`, that should not work anymore: `this.selectedKind = '';`